templates/supervisord.conf.j2

```diff
@@ -23,7 +23,7 @@ username = {{ supervisor_user }}
 password = {SHA}{{ supervisor_password|hash('sha1') }}
 
 [supervisorctl]
-serverurl = http://localhost:{{ supervisor_inet_http_server_port }}
+serverurl = {{ supervisor_inet_http_server_port }}
 username = {{ supervisor_user }}
 password = {SHA}{{ supervisor_password|hash('sha1') }}
```

Comment:
As of the supervisor documentation the password needs to be stated in plain text here.

```
The password to pass to the supervisord server for use in authentication. This should be the cleartext version of password from the supervisord server configuration for the port or UNIX domain socket you’re attempting to access. This value cannot be passed as a SHA hash. Unlike other passwords specified in this file, it must be provided in cleartext.
```

http://supervisord.org/configuration.html#supervisorctl-section-values